demisto_sdk/commands/generate_modeling_rules/RAEDME.md

```diff
@@ -0,0 +1,52 @@
+# `generate-modeling-rules`
+
+**Usage**:
+
+```console
+$ generate-modeling-rules [OPTIONS]
+```
+
+**Options**:
+
+* `-m, --mapping PATH`: The Path to a csv or tsv file containing the mapping for the modeling rules.  [required]
+* `-re, --raw_event PATH`: The path to a raw event from the api call in a json format.  [required]
+* `-dm, --data_model PATH`: The path to The one data model schema please for more information look at.  [required]
```

Comment:
I don't understand this description.
Is this a sample of the schema?
Or is there supposed to be a link to something else?

If this is a sample, then the text should be: The path to the one data model sample schema. 
If this is supposed to be a link to something, then the text should be, The path to the one data model schema. For more information, see ???

Reply:
the information is missing you are right will be fixed.